recipes/descwl-shear-sims/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "descwl-shear-sims" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LSSTDESC/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5c99d9cd8c6714d95e9f49232a0c10283c19ecc2ff2c3c1138818fafd5b868c5
+
+
+build:
+  number: 0
+  skip: True  # [py<310 or py>311 or win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - setuptools >=38
+    - pip
+  run:
+    - python
+    - numba
+    - galsim
+    - lsstdesc.weaklensingdeblending
+    - stackvana
```

Comment:
Having this in run would exclude someone using a stack not in conda-forge. We'll need to think about how to handle that.

Reply:
Conda won't recognize that.

Reply:
That is good for now. I think maybe I will add an external build of stackvana that is a shell package that folks can use against a custom DM env.